src/pages/components/utils/index.ts

```diff
@@ -22,6 +22,10 @@ export const bySeverityHandler = (a: HourlyData, b: HourlyData) => {
   return b.planned_rides - a.planned_rides
 }
 
+export const sortByMode = (hourlyData: HourlyData[] = [], sortingMode: string) => {
```

Comment:
why not using `hourlyData.sort(byHourHandler)` and `hourlyData.sort(bySeverityHandler)`  when needed?

`bySeverityHandler` should be refactored to `SeverityComperator`
`byHourHandler` should be refactored as well

Reply:
i prefer not to mutate hourlyData array (because sort do the mutation of array)
instead im using `toSorted` which is doing the same only it returns a sorted copy of array

Reply:
allright,
nevertheless, the usage of strings isn't very nice